File: src/prefix-sum/1480.cpp
class Solution
{
public:
  vector<int> runningSum(vector<int> &nums)
  {
    vector<int> prefixSums;
    prefixSums.push_back(nums[0]);

    for (int i = 1; i < nums.size(); i++)
    {
      prefixSums.push_back(prefixSums[i - 1] + nums[i]);
    }

    return prefixSums;
  }
};

File: src/prefix-sum/1480.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var runningSum = function (nums) {
  const prefixSums = [nums[0]];

  for (let i = 1; i < nums.length; i++) {
    prefixSums.push(prefixSums[i - 1] + nums[i]);
  }
  return prefixSums;
};

console.log(runningSum([1, 2, 3, 4]));
